frontend/src/pages/TermPlanner/OptionsHeader/OptionsHeader.tsx

```diff
@@ -106,7 +116,7 @@ const OptionsHeader = ({ plannerRef }: Props) => {
             <Popconfirm
               placement="bottomRight"
               title="Are you sure you want to unplan all your courses?"
-              onConfirm={() => dispatch(unscheduleAll())}
+              onConfirm={handleUnscheduleAll}
```

Comment:
I'm fairly sure this was the only use of the `unscheduleAll` reducer, so I think it should be fine to remove it from `plannerSlice.ts`?

Reply:
lgtm; this can easily be a one-off call

Reply:
@leonardo-fan does having it as an anonymous function vs passing the function in have any consequence with the Event object being passed in incidentally later on?

Reply:
nope this seems all good, handleUnscheduleAll is an anon function anyway.